pages/_app.js

```diff
@@ -42,6 +42,11 @@ export default class HomePage extends App {
             name="viewport"
             content="width=device-width, initial-scale=1.0"
           />
+          <link
+            rel="shortcut icon"
+            type="image/x-icon"
+            href="../static/imgs/favicon.ico"
+          />
```

Comment:
Although it works perfectly in an app, including the favicon in the _document file is the best option. The _app and _document have very similar functionalities, which include adjustments common to all pages, but there are small differences between them. _app is mostly used to apply shared layout and styles between pages, while document is used for metadata common to all pages. More details: 
https://nextjs.org/docs/pages/building-your-application/routing/custom-app
https://nextjs.org/docs/pages/building-your-application/routing/custom-document